data/apis/api3/beacons/coingecko btc_usd 0.1 percent deviation.json

```diff
@@ -8,7 +8,7 @@
   "chains": [
     {
       "active": true,
-      "name": "ropsten-2",
+      "name": "ropsten",
```

Comment:
I've removed this? Let me know if this is alright

Reply:
Should be fine. It was removed in another PR, so the two shouldn't conflict.